air quality and covid notebook

kathrynberger opened the pull request:
In this notebook I am attempting to walk through the VEDA Discovery [Air Quality and COVID-19 ](https://www.earthdata.nasa.gov/dashboard/covid-19/discoveries/air-quality-and-covid-19). In particular, I'm trying to showcase the side-by-side differences in the section "Seeing Rebounds in NO2" at the bottom of the Discovery. 

The data appears to match (Feb 2020 vs. Feb 2022) for area explored (China). Graphic on Dashboard Discovery story appears to match `nb` output in terms of data and rescale values (i.e., they were note re-scaled as we used Raster API). 

I've used a color scheme that best represents the data shown in the Dashboard story (using blues and purples, `cool`), however, I could not find the exact color spectrum used (as `rdbu_r` as I imagined was being used didn't show the same contrast).

jsignell commented on example-notebooks/air-quality-covid.ipynb:
```diff
@@ -0,0 +1,500 @@
+{
```
<p>This cell could use some sort of description.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

kathrynberger replied:
<p>I've added some clarifying text before this to give context. I haven't, however, said anything about the unit here (mainly because the catalog itself doesn't say anything about it - I'd imagine it is parts per million or similar. Not sure if that level of detail is required, but might help. </p>

jsignell commented on example-notebooks/air-quality-covid.ipynb:
```diff
@@ -0,0 +1,500 @@
+{
```
<p>nice!!!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

jsignell reviewed (approved):
This notebook looks great! I just have one suggestion for adding some additional context, but I'd be fine to merge as is. We should include it in the TOC too right?

kathrynberger commented:
I've added the nb to the ToC and added clarifying text where we discussed. Happy for you to just take a look at that revised line of text before merging @jsignell
